Sources/TokamakDOM/Views/Buttons/Button.swift

```diff
@@ -17,10 +17,25 @@
 
 import TokamakCore
 
-extension Button: ViewDeferredToRenderer where Label == Text {
+extension _Button: ViewDeferredToRenderer where Label == Text {
   public var deferredBody: AnyView {
-    AnyView(HTML("button", listeners: ["click": { _ in _ButtonProxy(self).action() }]) {
-      _ButtonProxy(self).label.subject
+    AnyView(HTML("button", listeners: [
```

Comment:
If the style is not `Default` it should set the `appearance` style to `none` to disable the button look.